scripts/add_existing_baseyear.py

```diff
@@ -616,6 +619,9 @@ def add_heating_capacities_installed_before_baseyear(
 
     # check if the CHPs were read in from MaStR for Germany
     if "Capacity_thermal" in chp.columns:
+        if "I" not in snakemake.wildcards.sector_opts.split("-"):
+            chp.query("Industry == False", inplace=True)
```

Comment:
```suggestion
            chp.query("not Industry", inplace=True)
```
this should also work.